src/Components/Aspire.Azure.Data.Tables/ConfigurationSchema.json

```diff
@@ -110,7 +110,7 @@
                             "NetworkTimeout": {
                               "type": "string",
                               "pattern": "^-?(\\d{1,7}|((\\d{1,7}[\\.:])?(([01]?\\d|2[0-3]):[0-5]?\\d|([01]?\\d|2[0-3]):[0-5]?\\d:[0-5]?\\d)(\\.\\d{1,7})?))$",
-                              "description": "The timeout applied to an individual network operations."
+                              "description": "The timeout applied to individual network operations."
```

Comment:
These came from

https://github.com/Azure/azure-sdk-for-net/commit/41ee2f026862c87bacca1219e45eaab2709ae00c